source/documentation/deploying_services/route_services.md

```diff
@@ -0,0 +1,108 @@
+## Route services
+
+Tenants may wish to apply some processing to requests before they reach an application. Common examples of use cases are authentication, rate limiting, and caching services.
+
+Cloud Foundry allows the tenants to bind [application routes](/#names-routes-and-domains) to [Route Services](https://docs.cloudfoundry.org/services/route-services.html) [external-link]. A Route Service acts as a full proxy. Once it is bound to a route, the platform routing layer will send every request for that route to the Route Service endpoint. The Route Service can then process the incoming request, proxy it back to the original application, and finally process the response request before returning it to the original client.
+
+Using Route Services has some consequences to be aware of:
+
+- Every request will be penalised with some additional latency as it will be proxied via the Routing Service.
+- The Route Service will be able to access all the request content in clear.
+- The Route Service would become a critical point of failure, and if it is not available, the application will not be available for the end users.
+
+### User-Provided Route Services
+
+Tenants can define their own Route Service instance by using a [User-Provided Service Instance](/#user-provided-service-instance) that points to any HTTPS service. This endpoint must fulfill the following requirements:
+
+- It must be a HTTPS endpoint with a valid certificate.
+- It can be a application running in the platform itself or a external service in the Internet.
```

Comment:
..."or **an** external service running **on** the internet" - notes in bold are changes, not text that should be bold